lib/iris/tests/runner/_runner.py

```diff
@@ -163,7 +163,7 @@ def run(self):
 
         args = ['', None, '--processes=%s' % n_processors,
                 '--verbosity=2', regexp_pat,
-                '--process-timeout=600']
+                '--process-timeout=180']
```

Comment:
@dkillick This change brought the `TEST_TARGET=default` matrix runners from `+18m` down to `6m17s` for `py27`, and `7m48s` for `py34` ... that's at least **10 minutes** quicker than I've been seeing for a while ... :tada: